### ISSUE TITLE
Accessibility: button background has insufficient contrast to pass WCAG 2.0 level AAA

### ISSUE THREAD

**Issue by laurelfulford** [Automattic/republication-tracker-tool#133]
The button and text don't have sufficient contrast to pass pass WCAG 2.0 level AAA -- the contrast ratio should be at least 4.5:1 for the larger font size. 

The current contrast ratio is 3.01:1 for #fffff against #5499DB:

![image](https://user-images.githubusercontent.com/177561/163028964-b4a77792-122f-487e-ab1d-541a6d0d5019.png)

Switching it to a darker blue (like #2978C2, contrast ratio 4.6:1) will fix this issue:

![image](https://user-images.githubusercontent.com/177561/163029055-8367b4e2-cce2-4d64-8b57-56bd368c4dc3.png)

Resource: https://webaim.org/resources/contrastchecker/

**Comment by laurelfulford** [comment]
#fff on #2A7AC2 is actually an exact 4.5:1 contrast.
